### ISSUE TITLE
Unmaintained `serde_cbor`

### ISSUE THREAD

**Issue by pinkforest** [dalek-cryptography/curve25519-dalek#431]
Crate `serder_cbor` is unmaintained

```
warning[A003]: serde_cbor is unmaintained
   ┌─ /home/foobar/pp/curve25519-dalek/Cargo.lock:60:1
   │
60 │ serde_cbor 0.11.2 registry+https://github.com/rust-lang/crates.io-index
   │ ----------------------------------------------------------------------- unmaintained advisory detected
   │
   = ID: RUSTSEC-2021-0127
   = Advisory: https://rustsec.org/advisories/RUSTSEC-2021-0127
   = The `serde_cbor` crate is unmaintained. The author has archived the github repository.
     
     Alternatives proposed by the author:
     
      * [`ciborium`](https://crates.io/crates/ciborium)
      * [`minicbor`](https://crates.io/crates/minicbor)
   = Announcement: https://github.com/pyfisch/cbor
   = Solution: No safe upgrade is available!
   = serde_cbor v0.11.2
     └── criterion v0.3.6
         └── (dev) curve25519-dalek v4.0.0-pre.2
```

**Comment by pinkforest** [comment]
This also comes via `criterion` 0.3.6 and 0.4.0 criterion has switched to `ciborium`
This will be solved by bumping to `criterion` 0.4.0

**Comment by rozbb** [comment]
Closed by #432
